src/MessageDispatch.KurrentDB/KurrentDbSubscriber.cs

```
@@ -107,6 +108,7 @@ private void PerformKeepAliveRead(object source, ElapsedEventArgs e)
             _kurrentDbClient.ReadStreamAsync(Direction.Backwards, _streamName, StreamPosition.End, maxCount: 1);

```

Probably best to wrap this in try/catch, log+swallow the exception if it fails.